### ISSUE TITLE
Clockface 4.0: Pickers - Color, Date Range, Date [in progress]

### ISSUE THREAD

**Issue by taramk** [influxdata/clockface#837]

**Comment by taramk** [comment]
@juliajames12 did you say you were doing work with the date range picker (time range?) or should i pick these up to design/recolor?

**Comment by juliajames12** [comment]
Yeah! Arie and I are working on a new time range component that we're going to test out in the new Script editor first with plans to implement everywhere.

**Comment by juliajames12** [comment]
Can we update the name of "Date Range Picker" to "Time Range Picker" as a part of this ticket? Small thing, but I think "Time Range Picker" is a more accurate name for it.

**Comment by taramk** [comment]
@juliajames12 - I'd probably add that to a list of improvements that we make after we get the recoloring done, because it would involve updates in the UI codebase as well (renaming it everywhere it's used), not just Clockface.
